src/components/TF2/Player/Player.tsx

```diff
@@ -189,6 +190,11 @@ const Player = ({
             // Immediately update local instance
             // Causes new info to immediately show
             player.localVerdict = e.toString();
+            color = displayColor(playerColors!, player, cheatersInLobby);
+            const newColour = color ? `background-color: ${color}` : '';

```

Comment:
Minor nitpick: inconsistent spelling of color vs colour

Reply:
Samm vs me spelling